jenkins.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+delete_app_version() {
+  yes | gcloud --project=$GOOGLE_PROJECT_ID \
```

Comment:
https://google.github.io/styleguide/shell.xml
Quote your variables and prefer "${GOOGLE_PROJECT_ID}"